modules/docs/arrow-docs/docs/docs/datatypes/validated/README.md

```diff
@@ -214,6 +214,79 @@ val houseNumber = config.parse(Read.intRead, "house_number").withEither { either
 houseNumber
 ```
 
+## Abstracting aways validation strategies with `ApplicativeError`
```

Comment:
Alternative validation strategies to `Validated`: using `ApplicativeError`? It's a bit more specific than `abstracting away`. It'd be worthwhile to have a version of this example on the ApplicativeError docs!